src/PartEditor/PartEditor.ts

```diff
@@ -114,6 +114,10 @@ export class PartEditorProvider implements vscode.CustomTextEditorProvider, Part
           console.log('selectByGraph by', message.backend);
           this.handleSelectByGraph(message.backend);
           return;
+
+        case 'updateBackend':
+          this.handleUpdateBackend(message.backend);
+          return true;
```

Comment:
other `case` just returns but this return `true`. Could you explain the reason a bit?

Reply:
typo :)
I'll remove `true`